### PR TITLE
docs/user/power/install_upi: 2 compute and copy-edits

### PR DESCRIPTION
**Pull request by manojnkumar**
Changes recommended from the original review comments in PR #3454 .

Change worker machine minimum to 2.
Reword air-gapped to restricted network install.
etc.

### CODE REVIEW

File: docs/user/power/install_upi.md [hunk 1]
```diff
@@ -20,15 +20,15 @@ The steps for performing a UPI-based install are outlined here. Example automati
...
 
-The smallest OpenShift 4.x clusters require the following host:
+The smallest OpenShift 4.x cluster requires the following hosts:
 
 * 1 bootstrap machine.
 
 * 3 control plane machines.
 
-* atleast 1 worker machine.
+* 2 worker machines.
```
Comment (bpradipt) [inline docs/user/power/install_upi.md:29]: @manojnkumar please take a look at this comment. https://github.com/openshift/installer/pull/3454#discussion_r411662675
1 worker node is fine and the minimum required. However lets go with what @wking recommends

Comment (manojnkumar) [reply docs/user/power/install_upi.md:29]: @wking : Thoughts on the 1 worker vs 2 worker minimum?

Comment (wking) [reply docs/user/power/install_upi.md:29]: openshift-docs say 2

**Comment by abhinavdahiya** [comment]
doc only update

**Pull request merged**